tests/ops/qubit/test_parametric_ops.py

```diff
@@ -1151,7 +1151,7 @@ def test_PauliRot_decomposition_XIYZ(self):
         assert decomp_ops[4].wires == Wires([2])
         assert decomp_ops[4].data[0] == -np.pi / 2
 
-    @pytest.mark.parametrize("angle", np.linspace(0, 2 * np.pi, 7))
+    @pytest.mark.parametrize("angle", npp.linspace(0, 2 * np.pi, 7, requires_grad=True))
```

Comment:
Was autograd numpy being imported as `npp` here? 😆 That's the 4th permutation I think I've seen (alongside `pnp`, `anp`, and `np`), we really should attempt to standardize it

Reply:
So true 😅 alongside `onp`/`np` variants for regular numpy.